src/test/java/automationpractice/com/pages/AboutUsPage.java

```diff
@@ -0,0 +1,16 @@
+package automationpractice.com.pages;
+
+import net.serenitybdd.core.pages.PageObject;
+import net.serenitybdd.core.pages.WebElementFacade;
+import org.openqa.selenium.support.FindBy;
+
+public class AboutUsPage extends PageObject {
+
+    @FindBy(xpath = "//*[contains(@class,'navigation_page') and contains(text(), 'About us')]")
+    private WebElementFacade AboutUsText;
+
+    public boolean isAboutUsTextDisplayed() {
+        return AboutUsText.isVisible();
+    }
+}
+
```

Comment:
O tym juz pisałam wyżej. Plus AboutUsText powinno byc z małej litery

Reply:
jeśli tytuły tych pagów się troszke zmienią to będziesz musiał to je przepisywać w 11 klasach w footerPages. Może stwórz sobie jakiś plik - klase albo enuma w data gdzie będziesz przetrzymywał tytuły tych stron

Reply:
tu chyba bede potrzebowal pomocy albo bede musial poszukac w necie jak to zrobic